node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsResponse.java

```diff
@@ -107,7 +108,10 @@ private void consumeMetricsItem(Inspector item, ListMap<String, Double> values)
         item.field("values").traverse((ObjectTraverser)(name, value) -> values.put(name, value.asDouble()));
     }
 
-    private boolean clusterIsStable(Node node, NodeList applicationNodes) {
+    private boolean clusterIsStable(Node node, NodeList applicationNodes, ListMap<String, Double> nodeValues) {
+        if (Metric.redistributing.from(nodeValues) > 0) {
```

Comment:
We should probably rely only on this?

Reply:
Yes, but I don't think the merge pending metric will be exposed in the "autoscale" set until the application upgrades?

Reply:
Good point!